cmd/exec/exec.go

```diff
@@ -21,30 +25,29 @@ func ExecutePlan(commands ...string) {
 	executionPlan.Print()
 
 	if executionPlan.HasBefore() {
-		executeAndStopIfFailed(executionPlan.Before)
+		executeAndStopIfFailed(executionPlan.Before, executeStart)
 	}
 
 	if executionPlan.HasCommands() {
 		for _, commandContext := range executionPlan.Commands {
-			executeAndStopIfFailed(commandContext)
+			executeAndStopIfFailed(commandContext, executeStart)
 		}
 	}
 
 	if executionPlan.HasAfter() {
-		executeAndStopIfFailed(executionPlan.After)
+		executeAndStopIfFailed(executionPlan.After, executeStart)
 	}
 
-	fmt.Println()
-	fmt.Println(utils.ColoredB("SUCCESS", utils.CYAN))
-
+	utils.PrintResultsBanner(true, executeStart)
 }
 
-func executeAndStopIfFailed(command *models.CommandContext) {
+func executeAndStopIfFailed(command *models.CommandContext, executeStart time.Time) {
 	command.PrintBanner()
 	err := command.CommandSession.Run()
 	if err != nil {
 		exitCode := (err.Error())[12:]
```

Comment:
The only thing missing is removing the `dashed banner` from the failure message.
<img width="1420" alt="Screenshot 2019-10-01 at 17 43 13" src="https://user-images.githubusercontent.com/10210778/65978050-4c100100-e473-11e9-9ced-b644ec52cf9c.png">

Reply:
Check issue description https://github.com/gopinath-langote/1build/issues/155 for more details.

Reply:
With message `Execution failed: phase 'a' - exit code: 127`